packages/apollo/src/plugin/plugin.spec.js

```diff
@@ -5,27 +5,58 @@ const { createPlugin: createPromsterMetricsPlugin } = require('./plugin');
 const {
   createServer: createPrometheusMetricsServer,
 } = require('@promster/server');
+const utils = require('@graphql-tools/utils');
+const schema = require('@graphql-tools/schema');
+
+const { makeExecutableSchema } = schema;
+const { getDirective, mapSchema, MapperKind } = utils;
+
+function throwErrorDirectiveTransformer(schema, directiveName = 'error') {
+  return mapSchema(schema, {
+    [MapperKind.OBJECT_FIELD]: (fieldConfig) => {
+      const errorDirective = getDirective(
+        schema,
+        fieldConfig,
+        directiveName
+      )?.[0];
```

Comment:
For readability I would not derefernce here. We can just add another firstDirective variable maybe which we either assign or destructure to.